bpf/lib/common.h

```diff
@@ -748,7 +748,7 @@ struct lb6_backend {
 	union v6addr address;
 	__be16 port;
 	__u8 proto;
-	__u8 pad;
+	__u8 flags;
```

Comment:
Should this be called `state`. 

IMO 'flags" makes me think this field is used for more then one config/purpose. But I think the usage just evaluates this field as the integer representing the backend state. Maybe better to just call it what it is.